create-aliquot.avsc

```diff
@@ -0,0 +1,79 @@
+{
+  "type": "record",
+  "name": "CreateAliquot",
+  "namespace": "uk.ac.sanger.psd",
+  "fields": [
+    {
+      "name": "id_lims",
+      "type": "string",
+      "doc": "The LIMS system that the aliquot was created in"
+    },
+    {
+      "name": "lims_uuid",
+      "type": "version4MessageUuid",
```

Comment:
Avro has a [`UUID`](https://avro.apache.org/docs/1.11.1/specification/_print/#uuid) logical type as well? But I'm not sure whether if it's possible to enforce v4 on it.

Reply:
It could be interesting to look at that.  I think I considered it and bypassed it in the past because it a) reads in the documentation like it will generate the UUID for you (though I don't think that's the case) and b) I may have tried to create messages with Avro in the early days where I couldn't get it to encode.  It may even be that `fastavro` doesn't support that logical type, which would have made me not use it.